modules/document_repository/README.md

```diff
@@ -0,0 +1,35 @@
+# Document Repository
+
+## Purpose
+
+ The Document Repository is a tool that provides studies with a centralized location for non-candidate documents. Files may be uploaded and organized under any user-defined category.
+
+## Intended Users
+ 
+ All Loris user and data entry staff could view, update, edit and delete the upload files with right permissions.
```

Comment:
I would edit to say: 
"Loris users can view, download and upload files, edit information about files, and delete files given appropriate permissions.
This module is used primarily by study coordinators/administrators who share study protocols and manuals, and by data entry staff who consult these documents. "